package.json

```diff
@@ -95,15 +95,44 @@
   },
   "release": {
     "branch": "release",
+    "plugins": [
+      [
+        "@semantic-release/release-notes-generator",
+        {
+          "parserOpts": {
+            "noteKeywords": [
+              "BREAKING CHANGE",
+              "BREAKING CHANGES",
+              "BREAKING"
```

Comment:
- would `BREAKING` not encompass the other two?
- is it worthwhile to check lowercase? or is it not case-sensitive?

Reply:
a) Good question.. I'm not sure offhand how they are doing the checking for this. Possibly overkill?

b) Only not checking because the uppercase is important. We should be YELLING this if it happens. This also allows `breaking` to be used in normal sentences in PRs without trigger an actual breaking change.